pages/pipelines/functions.rst

```diff
@@ -79,6 +79,8 @@ In our case the alternative version of calling ``parse_date`` would look like th
         set_field("transaction_year", new_date.year);
     end
 
+More examples of the usage of various functions in pipeline rules can be found in the `graylog2-server repo <https://github.com/Graylog2/graylog2-server/tree/master/graylog2-server/src/test/resources/org/graylog/plugins/pipelineprocessor/functions>`_.
```

Comment:
pointing to the master looks first like a good idea - but it will confuse if the master is way ahead than the current stable release. 

Like it is currently with the master holding 3.0 but stable is 2.4 - just use the link to the current release and this is fine.

Reply:
@jalogisch maybe it's my inexperience with git and stuff, but I switched to the branch 2.4 and that folder isn't there. You or @joschi know where I can get the same folder but on 2.4 specifically?

Reply:
Ah, thanks!